src/target/riscv/batch.c

```diff
@@ -132,18 +132,22 @@ static void add_idle_before_batch(const struct riscv_batch *batch, size_t start_
 }
 
 static int get_delay(const struct riscv_batch *batch, size_t scan_idx,
-		const struct riscv_scan_delays *delays)
+		const struct riscv_scan_delays *delays, bool resets_delays,
+		size_t reset_delays_after)
```

Comment:
1. `reset_delays` and `reset_delays_after`  are used for testing purposes only. IMHO, they should be removed one day, together with `riscv reset_delays` command.

Reply:
It sounds OK to me to remove this testing-only feature one day.

However, it would be good to replace it by a target that allows to test various command durations (DMI access, abstract command, system bus). I have been thinking about writing a simple target mock for that purpose, that would speak remote_bitbang protocol, similarly to Spike. However, I do not expect to find the time for that anytime soon :/